### PR TITLE
[DOCUMENTATION] Clean some documents how to for some James features

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

File: docs/modules/concepts/pages/protocols/jmap.adoc [hunk 1]
```diff
@@ -1,4 +1,29 @@
 = JSON Meta Application Protocol (JMAP)
 :navtitle: JMAP
 
-(TODO)
+== Overview
+
+https://en.wikipedia.org/wiki/Local_Mail_Transfer_Protocol[Local Mail Transfer Protocol] ("JMAP")
+is
+
+From <<RFC8620>>:
```
Comment (quantranhong1999) [inline docs/modules/concepts/pages/protocols/jmap.adoc:9]: blank line seems weird to me.

File: src/homepage/howTo/custom-listeners.html [hunk 2]
```diff
@@ -121,65 +112,14 @@ <h3><b>Starting James with BigMessageListener</b></h3>
...
-                <p>
-                    When you are using your listeners, place the jar containing your listeners into "extensions-jars" directory
-                    and add a volume for it.
-                </p>
-                <pre><code>
-volumes:
-  - $PWD/extensions-jars:/root/extensions-jars/
-                </code></pre>
-                <p>
-                    Putting compiled jars into extensions-jars directory
```
Comment (Arsnael) [inline src/homepage/howTo/custom-listeners.html:151]: Why removing the section explaining how to build and put custom jar into James?

Comment (vttranlina) [reply src/homepage/howTo/custom-listeners.html:151]: I added it above 
```
We need to be using the <code>listeners.xml</code> configuration that we had been writing and position the jar in the extensions-jars folder (specific to guice).```
```
And added the docker command (replace for docker-compose example)
```
docker run -p "25:25" -p "143:143" \
               -v "$PWD/src/main/resources/listeners.xml:/root/conf/listeners.xml" \
               -v "$PWD/target/custom-listeners:/root/extensions-jars/custom-listeners" \
        apache/james:memory-latest --generate-keystore
```

Make it simpler, and similar to other howTo documents (e.g.: custom-mailets, custom-healthchecks...)

File: docs/modules/concepts/pages/protocols/jmap.adoc (hunk 1 above)
Comment (vttranlina) [reply docs/modules/concepts/pages/protocols/jmap.adoc:9]: It normal, 
I copied format from other file
image

**Review by Arsnael — APPROVED** [review]

File: docs/modules/community/pages/download.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
 = Download Apache James releases
 :navtitle: Download releases
 
-(TODO)
+https://github.com/apache/james-project/tags[Download Apache James releases]
```
Comment (chibenwa) [inline docs/modules/community/pages/download.adoc:4]: Link to https://james.apache.org/download

File: examples/custom-healthcheck/README.md [hunk 4]
```diff
@@ -48,7 +42,7 @@ $ docker run -d \
    -v $PWD/healthcheck-extension.jar:/root/extensions-jars \
    -v $PWD/keystore:/root/conf/keystore \
    -p 25:25 \
-   apache/james:memory-latest
+   apache/james:memory-latest --generate-keystore
```
Comment (chibenwa) [inline examples/custom-healthcheck/README.md:45]: Then remove keystore volume?

File: examples/custom-smtp-hooks/README.md [hunk 5]
```diff
@@ -103,7 +97,7 @@ docker run -d \
   -v $PWD/exts:/root/extensions-jars \
   -v $PWD/keystore:/root/conf/keystore \
   -p 25:25 \
-  apache/james:memory-latest
+  apache/james:memory-latest --generate-keystore
```
Comment (chibenwa) [inline examples/custom-smtp-hooks/README.md:100]: Idem

File: examples/custom-webadmin-route/README.md [hunk 6]
```diff
@@ -68,7 +62,7 @@ $ docker run -d \
    -v $PWD/exts:/root/extensions-jars \
    -v $PWD/keystore:/root/conf/keystore \
    -p 25:25 \
-   apache/james:memory-latest
+   apache/james:memory-latest --generate-keystore
```
Comment (chibenwa) [inline examples/custom-webadmin-route/README.md:65]: Idem remove the mount for password

File: src/homepage/howTo/custom-healthchecks.html [hunk 7]
```diff
@@ -67,18 +67,14 @@ <h1><b>Configure Custom Healthchecks</b></h1>
...
-
                 <p>Then start a James server with your JAR and the configuration:</p>
 
                 <pre><code>docker run -d \
    -v $PWD/healthcheck.properties:/root/conf/healthcheck.properties \
    -v $PWD/healthcheck-extension.jar:/root/extensions-jars \
    -v $PWD/keystore:/root/conf/keystore \
    -p 25:25 \
-   apache/james:memory-latest</code></pre>
+   apache/james:memory-latest --generate-keystore</code></pre>
```
Comment (chibenwa) [inline src/homepage/howTo/custom-healthchecks.html:77]: Remove the volume for keystore

File: src/homepage/howTo/custom-smtp-hooks.html [hunk 8]
```diff
@@ -126,18 +126,14 @@ <h2><b>Writing your own hooks</b></h2>
...
-
                 <p>Then start a James server with your JAR and the configuration:</p>
 
                 <pre><code>docker run -d \
   -v $PWD/smtpserver.xml:/root/conf/smtpserver.xml \
   -v $PWD/exts:/root/extensions-jars \
   -v $PWD/keystore:/root/conf/keystore \
   -p 25:25 \
-  apache/james:memory-latest
+  apache/james:memory-latest --generate-keystore
```
Comment (chibenwa) [inline src/homepage/howTo/custom-smtp-hooks.html:136]: Idem

File: src/homepage/howTo/custom-webadmin-routes.html [hunk 9]
```diff
@@ -85,18 +85,14 @@ <h1><b>Configure Custom WebAdmin routes</b></h1>
...
-
                 <p>Then start a James server with your JAR and the configuration:</p>
 
                 <pre><code>docker run -d \
    -v $PWD/webadmin.properties:/root/conf/webadmin.properties \
    -v $PWD/exts:/root/extensions-jars \
    -v $PWD/keystore:/root/conf/keystore \
    -p 25:25 \
-   apache/james:memory-latest</code></pre>
+   apache/james:memory-latest --generate-keystore</code></pre>
```
Comment (chibenwa) [inline src/homepage/howTo/custom-webadmin-routes.html:95]: Idem

**Review by Arsnael — APPROVED** [review]

**Review by quantranhong1999 — APPROVED** [review]

**Comment by Arsnael** [comment]
Can you rebase and squash @vttranlina ?

**Comment by vttranlina** [comment]
rebase and squash

**Pull request merged**